cloudmetrics/metrics/fourier.py

```diff
@@ -0,0 +1,573 @@
+        else:
+            imin = imin[0]
+        if imin == imax:
+            psdi = psd_1d_rad[imin]
+        else:
+            psdi = psd_1d_rad[imin:imax]
+        psd_bin[i] = np.mean(psdi)
+
+    # Filter empty bins
+    bins_centres = bins_centres[psd_bin != 0]
+    psd_bin = psd_bin[psd_bin != 0]
+
+    return bins_centres, psd_bin
+
+
+def _debug_plot(
+    cloud_scalar,
+    k1d,
+    psd_1d,
+    b0,
+    beta,
+    r_squared_beta,
+    bins_centres,
+    psd_bins,
+    beta_binned,
+    b0_binned,
+    r_squared_binned,
+    l_spec_median,
+    l_spec_mom,
+):
+
+    fig, axs = plt.subplots(ncols=2, figsize=(8, 4))
+    axs[0].imshow(cloud_scalar, "gray")
+    axs[0].axis("off")
+    axs[0].set_title("Clouds")
+    axs[1].scatter(k1d, psd_1d, s=2.5, c="k")
+    axs[1].plot(k1d, b0 * k1d ** beta, c="k")
+    axs[1].scatter(bins_centres, psd_bins, s=2.5, c="C1")
+    axs[1].axvline(2 * np.pi / l_spec_mom, c="grey")
+
+    locs = axs[1].get_xticks().tolist()
+    labs = [x for x in axs[1].get_xticks()]
+    Dticks = dict(zip(locs, labs))
+    Dticks[2 * np.pi / l_spec_mom] = r"2pi/lspec"
+    locas = list(Dticks.keys())
+    labes = list(Dticks.values())
+    axs[1].set_xticks(locas)
+    axs[1].set_xticklabels(labes)
+
+    axs[1].plot(bins_centres, b0_binned * bins_centres ** beta_binned, c="C1")
+    axs[1].annotate("Direct", (0.8, 0.9), xycoords="axes fraction", fontsize=10)
+    axs[1].annotate(
+        r"$R^2$=" + str(round(r_squared_beta, 3)),
+        (0.8, 0.8),
+        xycoords="axes fraction",
+        fontsize=10,
+    )
+    axs[1].annotate(
+        r"$\beta=$" + str(round(beta, 3)),
+        (0.8, 0.7),
+        xycoords="axes fraction",
+        fontsize=10,
+    )
+    axs[1].annotate(
+        r"L (median) = " + str(round(l_spec_median, 3)),
+        (0.8, 0.6),
+        xycoords="axes fraction",
+        fontsize=10,
+    )
+    axs[1].annotate(
+        r"L (moment) =" + str(round(l_spec_mom, 3)),
+        (0.8, 0.5),
+        xycoords="axes fraction",
+        fontsize=10,
+    )
+    axs[1].annotate(
+        "Bin-averaged",
+        (0.4, 0.9),
+        xycoords="axes fraction",
+        color="C1",
+        fontsize=10,
+    )
+    axs[1].annotate(
+        r"$R^2$=" + str(round(r_squared_binned, 3)),
+        (0.4, 0.8),
+        xycoords="axes fraction",
+        color="C1",
+        fontsize=10,
+    )
+    axs[1].annotate(
+        r"$\beta_a=$" + str(round(beta_binned, 3)),
+        (0.4, 0.7),
+        xycoords="axes fraction",
+        color="C1",
+        fontsize=10,
+    )
+    axs[1].set_xscale("log")
+    axs[1].set_yscale("log")
+    axs[1].set_xlabel(r"$k$", fontsize=10)
+    axs[1].set_ylabel(r"$kF(C)^2$", fontsize=10)
+    axs[1].grid()
+    axs[1].set_title("1D radial pectrum")
+    plt.tight_layout()
+    plt.show()
+
+
+def compute_spectra(
+    cloud_scalar, dx=1, periodic_domain=False, apply_detrending=False, window=None
```

Comment:
In the doc-string you write that this is the "Cloud mask field" - is it a mask or a scalar 2D array? If it's a mask I would change the argument name to `cloud_mask`

Reply:
Looking at your test I'm quite sure this isn't the "cloud mask", so we should just change the doc-string then :)

Reply:
Should have checked the docstrings I copied over from an object metric before pushing this, sorry... Good catch!